tests/test_vscode_config.py

```
@@ -0,0 +1,234 @@
+        assert len(vscode_raw) < 10000, \
+            "Settings file seems excessively large (>10KB)"
+    
+    def test_no_sensitive_information(self, vscode_raw):
+        """Test that settings don't contain sensitive information"""
+        sensitive_patterns = ['password', 'token', 'api_key', 'secret', 'credential']
+        lower_content = vscode_raw.lower()
+        
+        for pattern in sensitive_patterns:
+            if pattern in lower_content:
+                # Check if it's just a setting name (key), not a value
+                lines = [l for l in vscode_raw.split('\n') if pattern in l.lower()]
```

Ruff (0.14.7)

Ambiguous variable name: `l`

(E741)